### ISSUE TITLE
Missing COCO Java dependency

### ISSUE THREAD

**Issue by GiovanniGabbolini** [ilaria-manco/muscaps#3]
The computation of the MS-COCO metrics fails due to a missing dependency.

# Steps to reproduce
By running `caption.py`, e.g. `python muscaps/scripts/caption.py your_experiment_id --metrics True` I got: 
```
Loading dataset
Building model
loading annotations into memory...
0:00:00.001473
creating index...
index created!
Loading and preparing results...     
DONE (t=0.00s)
creating index...
index created!
tokenization...
Error: Could not find or load main class edu.stanford.nlp.process.PTBTokenizer
Error: Could not find or load main class edu.stanford.nlp.process.PTBTokenizer
```

# My solution
In the original MS-COCO repository there's a [file](https://github.com/tylin/coco-caption/blob/master/pycocoevalcap/tokenizer/stanford-corenlp-3.4.1.jar ) that is missing in this repository: 
Downloading that file and copying it in `muscaps/caption_evaluation_tools/coco_caption/pycocoevalcap/tokenizer/` solved the issue for me.

**Comment by GiovanniGabbolini** [comment]
Follow-up: I encountered many more problems after the first one, mainly due to missing files.

I'll share here what I did:

-  Add [this](https://github.com/tylin/coco-caption/blob/master/pycocoevalcap/meteor/meteor-1.5.jar) file and [this](https://github.com/tylin/coco-caption/tree/master/pycocoevalcap/meteor/data) folder in `muscaps/caption_evaluation_tools/coco_caption/pycocoevalcap/meteor/`;
- Fix `meteor.py` as indicated by [here](https://github.com/tylin/coco-caption/issues/17#issuecomment-441460026_) @yhung119 ;
- Add [this](https://github.com/tylin/coco-caption/blob/master/pycocoevalcap/spice/spice-1.0.jar) file and [this](https://github.com/tylin/coco-caption/tree/master/pycocoevalcap/spice/lib) folder in `muscaps/caption_evaluation_tools/coco_caption/pycocoevalcap/spice/`;
- Run `sh muscaps/caption_evaluation_tools/coco_caption/get_stanford_models.sh`.

After all of this the code runs for me.

**Comment by ilaria-manco** [comment]
Hi Giovanni, thank you for reporting this. 

I have reproduced the issues and added all missing files as suggested, so the COCO evaluation code should now run without errors following pull request #4.

However I did not find the following proposed fix was necessary

> * Fix `meteor.py` as indicated by [here](https://github.com/tylin/coco-caption/issues/17#issuecomment-441460026_) @yhung119 

As for 

> * Run `sh muscaps/caption_evaluation_tools/coco_caption/get_stanford_models.sh`

that is indeed required and is already mentioned in the [`caption_evaluation_tool` README](https://github.com/ilaria-manco/muscaps/tree/main/muscaps/caption_evaluation_tools).

Thank you again for your help!
